IOMC/ParticleGuns/src/CloseByParticleGunProducer.cc

```diff
@@ -25,9 +25,12 @@ CloseByParticleGunProducer::CloseByParticleGunProducer(const ParameterSet& pset)
   ParameterSet defpset;
   ParameterSet pgun_params = pset.getParameter<ParameterSet>("PGunParameters");
 
+  fControlledByEta = pgun_params.getParameter<bool>("ControlledByEta");
   fEnMax = pgun_params.getParameter<double>("EnMax");
   fEnMin = pgun_params.getParameter<double>("EnMin");
   fMaxEnSpread = pgun_params.getParameter<bool>("MaxEnSpread");
+  fEtaMax = pgun_params.getParameter<double>("MinEta");
+  fEtaMin = pgun_params.getParameter<double>("MaxEta");
```

Comment:
@waredjeb Min is max and max is min here. Maybe they should be flipped? Also, could you please add a line in the description about the bug that PR #36460 introduced?

Reply:
Thanks! I fixed the names and added a description of the bug introduced